src/components/auth/PrivacyAwareSignInButton.tsx

```
@@ -77,6 +77,7 @@ export function PrivacyAwareSignInButton({
     });
   };
 
+
   const handlePrivacyConsentClose = () => {
```

ESLint

[error] 'handlePrivacyConsentClose' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)